oauthenticator/azuread.py

```diff
@@ -50,6 +65,23 @@ def _token_url_default(self):
             self.tenant_id
         )
 
+    @default('scope')
+    def _scope_default(self):
+        return ['openid']
+
+    role_claim = Unicode(config=True)
```

Comment:
This should be documented with a `help` string. I'm trying to review this code atm with little knowledge of AzureAD, so having a sentence or two to describe what something is about is critical.

Reply:
@consideRatio are you referring to the scope part or the role_claim part?

Reply:
This comment was about te `role_claim` part, you can pass `help="documentation here"` to the `Unicode()` constructor. It will end up in the documentation website then.

Reply:
Disregard.. it's obvious you mean the `role_claim`  trait.